doc/classes/Texture2DArray.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Texture2DArray" inherits="TextureLayered" version="4.0">
 	<brief_description>
+		An array of [Texture2D] resources to be used as a 3D texture.
```

Comment:
There is already a pending PR for this https://github.com/godotengine/godot/pull/36541

So can you just cut this please?